app/models/gws/staff_record/user.rb

```diff
@@ -0,0 +1,127 @@
+class Gws::StaffRecord::User
+  include SS::Document
+  include Gws::Referenceable
+  include Gws::Reference::User
+  include Gws::Reference::Site
+  include Gws::StaffRecord::Yearly
+  include Gws::Addon::ReadableSetting
+  include Gws::Addon::GroupPermission
+  include Gws::Addon::History
+  include Gws::Export
+
+  seqid :id
+  field :name, type: String
+  field :code, type: String
+  field :order, type: Integer, default: 0
+  field :kana, type: String
+  field :multi_section, type: String, default: 'regular'
+  field :section_name, type: String
+  field :section_order, type: Integer
+  field :title_name, type: String
+  field :tel_ext, type: String
+  field :charge_name, type: String
+  field :charge_address, type: String
+  field :charge_tel, type: String
+  field :divide_duties, type: String
+  field :remark, type: String
+  field :staff_records_view, type: String, default: 'show'
+  field :divide_duties_view, type: String, default: 'show'
+
+  permit_params :name, :code, :order, :kana, :multi_section, :section_name,
+                :title_name, :tel_ext, :charge_name, :charge_address, :charge_tel,
+                :divide_duties, :remark, :staff_records_view,  :divide_duties_view
+
+  validates :name, presence: true
+  validates :code, presence: true#, uniqueness: { scope: [:site_id, :year] }
+  validates :multi_section, inclusion: { in: %w(regular plural) }
+  validates :section_name, presence: true
+  validates :staff_records_view, inclusion: { in: %w(show hide) }
+  validates :divide_duties_view, inclusion: { in: %w(show hide) }
+
+  before_validation :set_section_order, if: -> { section_name.present? }
+
+  default_scope -> { order_by section_order: 1, section_name: 1, order: 1 }
```

Comment:
__[rails_best_practices]__
* default_scope is evil [:ref](http://rails-bestpractices.com/posts/2013/06/15/default_scope-is-evil/)

<!-- Comment by [SideCI](https://sideci.com) -->